cpp/inc/bond/core/reflection.h

```diff
@@ -101,24 +101,21 @@ struct FieldTemplate
     /// @brief Static data member describing field metadata
     static const Metadata& metadata;
 
-    /// @brief Static data member representing the field pointer
-    static const field_pointer field;
-
```

Comment:
I added it back for now, we may want to handle vc12 explicitly to remove this symbol from generated .obj files.